zer0-os/zOS
# feat: add edit message icon tooltips (save / discard)

### What does this do?
- adds tooltips to the edit message action icons.

### Why are we making this change?
- as per figma design.

### How do I test this?
- open messenger. hover over a message you have sent and click edit. hover over the 'x' icon to see 'discard changes', and hover over the check icon to see 'save changes'

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?



https://github.com/zer0-os/zOS/assets/39112648/98806753-32f9-4f1c-95de-021d9e652e32
